src/olympia/reviewers/tests/test_forms.py

```diff
@@ -95,7 +95,7 @@ def test_actions_addon_status_null(self):
         actions = self.get_form().helper.get_actions()
         assert list(actions.keys()) == [
             'unreject_latest_version',
-            'clear_pending_rejection_multiple_versions',
+            'change_pending_rejection_multiple_versions',
```

Comment:
nit: it is no longer clear in the name you can also "clear" via this action. Maybe "change_or_clear_pending_rejection_multiple_versions" is super long, but more clear? Or (for some reason) "set" "update" read a bit more broadly to me. Somehow from "change" i'm inferring that there is a value to change to.

Reply:
e1ffe0983d8264314fe1adbd8353815ef295e3ab

Kept the label the same to avoid wrapping, but updated (eh) the internal name, and the description to mention that changing the date will notify developers.